test/format/config.c

```diff
@@ -998,7 +998,13 @@ config_mirrors(void)
     for (i = 1; i <= ntables; ++i)
         if (tables[i]->type != FIX)
             break;
-    if (ntables < 2 || i > ntables) {
+
```

Comment:
Since mirror configuration is complex, maybe add a comment here.

Reply:
Good call, thanks for the reminder. Added a comment in ab8c36b.